src/openassetio-python/cmodule/src/EntityReferenceBinding.cpp

```diff
@@ -1,18 +1,23 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2013-2022 The Foundry Visionmongers Ltd
+#include <fmt/format.h>
 #include <pybind11/operators.h>
 #include <pybind11/pybind11.h>
 
 #include <openassetio/EntityReference.hpp>
 
 #include "_openassetio.hpp"
 
-void registerEntityReference(const py::module &mod) {
+void registerEntityReference(const py::module& mod) {
   using openassetio::EntityReference;
 
   py::class_<EntityReference>{mod, "EntityReference", py::is_final()}
       .def(py::init<openassetio::Str>(), py::arg("entityReferenceString"))
       .def("toString", &EntityReference::toString)
       .def("__str__", &EntityReference::toString)
+      .def("__repr__",
+           [](const EntityReference& self) {
+             return fmt::format("<openassetio.EntityReference {}>", self.toString());
```

Comment:
[consider][minor] Put the string in quotes? Don't mind whether single or double

Reply:
Had pondered that - but then do we need to escape any of which ever character we pick in the presentation?  Given its only for debug - opted to dodge that entire issue for now (as technically we should maybe be escaping `>` too?). Seem reasonable?